internal/events/eventhandlers/send_receiver_wallets_sms_invitation_event_handler.go

```diff
@@ -26,9 +27,10 @@ type SendReceiverWalletsSMSInvitationEventHandlerOptions struct {
 }
 
 type SendReceiverWalletsSMSInvitationEventHandler struct {
-	tenantManager      tenant.ManagerInterface
-	crashTrackerClient crashtracker.CrashTrackerClient
-	service            services.SendReceiverWalletInviteServiceInterface
+	tenantManager          tenant.ManagerInterface
+	tenantDBConnectionPool db.DBConnectionPool
```

Comment:
I think this naming would be a bit clearer:
```suggestion
	mtnDBConnectionPool db.DBConnectionPool
```

Similar to what's used in:
https://github.com/stellar/stellar-disbursement-platform-backend/blob/cc78370d0b2e2ddce59860a081522711ae76e5db/internal/serve/serve.go#L118-L121